azure/table_azure_network_watcher_flow_log.go

```diff
@@ -0,0 +1,222 @@
+package azure
+
+import (
+	"context"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-05-01/network"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+)
+
+type flowLogInfo = struct {
+	network.FlowLog
+	NetworkWatcherName string
+}
+
+//// TABLE DEFINITION
+
+func tableAzureNetworkWatcherFlowLog(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_network_watcher_flow_log",
+		Description: "Azure Network Watcher FlowLog",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.AllColumns([]string{"network_watcher_name", "name", "resource_group"}),
+			Hydrate:           getNetworkWatcherFlowLog,
+			ShouldIgnoreError: isNotFoundError([]string{"ResourceNotFound", "ResourceGroupNotFound", "404"}),
+		},
+		List: &plugin.ListConfig{
+			Hydrate:       listNetworkWatcherFlowLogs,
+			ParentHydrate: listNetworkWatchers,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The friendly name that identifies the flow log.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "id",
+				Description: "Contains ID to identify a flow log uniquely.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromGo(),
+			},
+			{
+				Name:        "enabled",
+				Description: "Indicates whether the flow log is enabled, or not.",
+				Type:        proto.ColumnType_BOOL,
+				Transform:   transform.FromField("FlowLogPropertiesFormat.Enabled"),
+			},
+			{
+				Name:        "network_watcher_name",
+				Description: "The friendly name that identifies the network watcher.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "etag",
+				Description: "An unique read-only string that changes whenever the resource is updated.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "type",
+				Description: "The resource type of the flow log.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "flowlog_format_type",
```

Comment:
Should this column name be reduced to `file_type`? There's no Terraform equivalent, but seems suitable based on the description.